log-warper.cabal

```diff
@@ -52,7 +52,7 @@ library
                      , fmt                  >= 0.5.0.0
                      , formatting           >= 6.2.2
                      , hashable             >= 1.2.4.0
-                     , lens                 >= 4.14
+                     , microlens-platform   >= 0.3.9
```

Comment:
Could you explain briefly why you needed `microlens-platform` (which combines `microlens`, `microlens-mtl`, `microlens-th` and `microlens-ghc`) instead of combination of those ones?

Reply:
It's because there is no instances for `Text` in `microlens-ghc`, they go only with `microlens-platform`, so as we are using `Text` with for example `at` function in `System.Wlog.LoggerConfig` module [`zoomLogger`](https://github.com/serokell/log-warper/blob/142b228979cc428eaea1fc27f73e0170d0c50e07/src/System/Wlog/LoggerConfig.hs#L187) function, it is not possible to live without `platform`

Reply:
Oh, I see... Well, `zoomLogger` function should be refactored. I think it has bad implementation because it takes `Text` instead of `LoggerName`. Could you please refactor this function to take `LoggerName` and also change `LoggerName` to this?

```haskell
type LoggerMap = HashMap LoggerName LoggerTree
```

After you do this you can implement `type family` instances for `LoggerName` and `LoggerMap` like it's done here: 

http://hackage.haskell.org/package/microlens-0.4.8.1/docs/src/Lens-Micro-Internal.html#Index

And when this change will be performed you can replace `microlens-platform` with 4 dependencies. I think it worth it because it will improve code quality and make `log-warper` a little bit lighter. You might not even need `microlens-ghc` I guess... But I'm not sure.

Reply:
I think `Index` and `IxValue` is not enough for this purpose, also `At` instance is needed for `Hashmap`.

Reply:
Sorry, you're totally right. I'm just blind. And I'm not an expert on `microlens-*` library family. It's modularized in some nontrivial way. Yes, in that case let keep `microlens-platform` in dependencies. And you don't need to implement type family instances manually. Only perform that refactoring with `LoggerName`.